packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -469,8 +469,12 @@ export const makeZCFZygote = async (
 
       await null;
       if (!zcfBaggage.has('repairedContractCompletionWatcher')) {
-        await E(zoeInstanceAdmin).repairContractCompletionWatcher();
-        console.log(`Repaired contract completion watcher`);
+        // We don't wait because it's a cross-vat call (to Zoe) that can't be
+        // completed during this vat's start-up
+        E(zoeInstanceAdmin)
+          .repairContractCompletionWatcher()
+          .catch(() => {});
```

Comment:
This is not strictly a testing change anymore, right?

Reply:
correct. 

I'd guess you're hinting that I should change something, presumably the commit logs, but I don't understand their impact. Is it sufficient that the latest commit has a `feat` tag, or do I need to change the PR's title or something else? Do I need to eventually need to check it in as a merge rather than a squash, or is there something else to be careful of?

Reply:
The main problem is that this change is in a commit titled `test:`. This change should be in a separate commit, before the 'test' commit, describing this change in behavior. 

The PR title says "test" but it's normal for other changes to be made to support a testing requirement. The problem is when the changelog leaves those changes out.

Reply:
The commit makes two changes. One is the fix to `startInstance.js`. The other is a change to the test probe to take advantage of the fix and demonstrate that it fixes the problem. That second change has to update the test, so it can't precede it. I think it's important that the fix and the revelation of its effectiveness stay together. 

If I amend the commit to call it a `fix`, and remember to not squish, would that be sufficient to update the changelog?  If not how would you recommend I address this?

Reply:
> I think it's important that the fix and the revelation of its effectiveness stay together.
Up to you.

>If I amend the commit to call it a fix, and remember to not squish, would that be sufficient to update the changelog?
Yes

Reply:
done

Reply:
Looks like it's still in a "test" commit.
https://github.com/Agoric/agoric-sdk/commit/d7ba6846fa1b3e228aac8975ff31e665d240e9d5#diff-8dcfebb96abffde204086c5d660142e0c4c74a7bac0800f7c18447e0baad1979R472-R477